packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -67,7 +63,7 @@ export interface QueueProcessingServiceBaseProps {
   readonly environment?: { [key: string]: string };
 
   /**
-   * Secret environment variables to pass to the container
+   * The secret environment variables to pass to the container
```

Comment:
Perhaps you also want a period for this.

Reply:
Updated